server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Capability.scala

```diff
@@ -34,6 +34,8 @@ object CapabilityIdentifier {
   val JMAP_MAIL: CapabilityIdentifier = "urn:ietf:params:jmap:mail"
   val JAMES_QUOTA: CapabilityIdentifier = "urn:apache:james:params:jmap:mail:quota"
   val JAMES_SHARES: CapabilityIdentifier = "urn:apache:james:params:jmap:mail:shares"
+
+  val ALL_CAPABILITIES: Set[CapabilityIdentifier] = Set(JMAP_CORE, JMAP_MAIL, JAMES_QUOTA, JAMES_SHARES)
```

Comment:
```suggestion
  val SUPPORTED_IDS: Set[CapabilityIdentifier] = Set(JMAP_CORE, JMAP_MAIL, JAMES_QUOTA, JAMES_SHARES)
```
?

Reply:
or `SUPPORTED_CAPABILITIES`? IDS sounds a bit too generic... IDs of what?

Reply:
That's already in an object call Capability, hence my suggestion.

BTW there is already a DefaultCapabilities object with a SUPPORTED field. So please complete this field and use it.

Reply:
no answer on this point? Else I agree with the other changes, thanks

Reply:
I contributed it here: https://github.com/linagora/james-project/pull/3613/commits/8496e74952d70881a8475bf4beeefe7083cdfbae

please cherry-pick my commit, then I :+1: with this PR